lib/signer.ex

```diff
@@ -40,78 +40,85 @@ defmodule AntikytheraAws.Signer do
     - `params` must be decoded query strings, in a list of tuple-2
         - To convert from `Antikythera.Http.QueryParams` (which is `Croma.SubtypeOfMap`), just use `Map.to_list/1`
 
-    Note: "content-type" and "x-amz-content-sha256" headers are required for S3, but not generally.
+    Note: "content-type" header is required for S3, but not generally.
```

Comment:
Concerning the doc comment before this PR's modifications...
Is "content-type" header really required?
At least for [put-object](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectPUT.html) it seems not mandatory.
You've already successfully uploaded a file without content-type, haven't you?

Reply:
Ah, sure. Yes, I certainly uploaded a file without using content-type.
Even `GET`, `POST`, and `DELETE` does not seem to be compulsory.
deleted in e495118